src/App.tsx

```
@@ -1,10 +1,106 @@
+  user = "user",
+  guest = "guest",
+}
+
+const SomENum = {
+  ADMIN: "admin",
+} as const;
+
+// ❌ More violations
+const usr = "john_doe"; // ❌ Abbreviation
+const cfg = { timeout: 5000 }; // ❌ Abbreviation + magic number + missing units
+const mgr = "manager"; // ❌ Abbreviation
```

ESLint

[error] 'mgr' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)